src/main/java/gui/GUIDriver.java

```diff
@@ -24,7 +24,7 @@
     @Override
     public void start(Stage primaryStage) throws Exception{
         configure();
-        Parent root = FXMLLoader.load(Objects.requireNonNull(getClass().getResource("/basicPage.fxml")));
+        Parent root = FXMLLoader.load(Objects.requireNonNull(getClass().getResource("/monthlyCalendar.fxml")));
```

Comment:
Are we going to change `monthlyCalendar.fxml` to the main page later?

Reply:
No it will be the `mainPage.fxml` in junru's pr, but since there isnt a main right now, I used the monthly one first.